perllib/FixMyStreet/App/Controller/Admin/ManifestTheme.pm

```diff
@@ -0,0 +1,97 @@
+package FixMyStreet::App::Controller::Admin::ManifestTheme;
+use Moose;
+use namespace::autoclean;
+
+BEGIN { extends 'Catalyst::Controller'; }
+
+use FixMyStreet::App::Form::ManifestTheme;
+
+sub auto :Private {
+    my ($self, $c) = @_;
+
+    if ( $c->cobrand->moniker =~ /^default|fixmystreet$/ ) {
```

Comment:
Also just to note that regex matches anything beginning with default or anything ending with fixmystreet :)